pytest_fixtures/component/taxonomy.py

```diff
@@ -110,27 +102,6 @@ def class_sca_manifest_org(class_org, class_sca_manifest, class_target_sat):
     return class_org
 
 
-@pytest.fixture(scope='module')
-def module_extra_rhel_entitlement_manifest_org(
-    module_target_sat,
-    module_org,
-    module_extra_rhel_entitlement_manifest,
-):
-    """Creates an organization and uploads an entitlement mode manifest generated by manifester and
-    containing more RHEL entitlements than the default entitlement manifest"""
-    module_org.sca_disable()
-    module_target_sat.upload_manifest(module_org.id, module_extra_rhel_entitlement_manifest.content)
-    return module_org
-
-
-@pytest.fixture
-def function_entitlement_manifest_org(function_org, function_entitlement_manifest, target_sat):
```

Comment:
this also shows up in search at a couple of places ...

Reply:
Those use cases are removed here. This PR needs to be merged first https://github.com/SatelliteQE/robottelo/pull/15637